doc/TESTS.rst.txt

```diff
@@ -124,6 +132,9 @@ at the bottom.
 Labeling tests with nose
 ------------------------
```

Comment:
I think "label" should be changed to "docorators", and we no longer use any from nose. What we do add are the functions in `numpy/testing/_private/utils.py`. Note that `pytest.mark.slow` and such are now in pytest. At least we did not need to do anything to get it in NumPy. I think SciPy did, but that was done earlier.

Basically, there is *no* nose dependency left in NumPy except for the (optional) testing extensions we maintain for downstream, which are simply not available is nose is not present.